Source/OxyPlot/Series/ContinuousHistogramItem.cs

```diff
@@ -0,0 +1,120 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ContinuousHistogramItem.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Represents an item in a RectangleSeries.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Series
+{
+    /// <summary>
+    /// Represents an item in a <see cref="ContinuousHistogramSeries" />, a bin (range) and its area.
+    /// </summary>
+    public class ContinuousHistogramItem
```

Comment:
do we need these items? Couldn't the continuous histogram series be defined simple by start, end and an array of values?

Reply:
Something needs to decide how to allocate the bins: this way the user has complete freedom (and we can provide methods to make it easy for, e.g. start/end/samples as presented). Not to mention that the user may not actually have the sample data, only counts collected in bins, in which case anything else would present a miserable challenge.